### PR TITLE
Mixing: Syntax and virtual control group info.

### PR DESCRIPTION
**Pull request by hamishwillee**
Fix for https://github.com/PX4/Devguide/issues/868 

Note, this fixes the specific problems listed in #868. That is not to say that the mixer documentation is now perfect. 

What it makes clear is:
1. Virtual mixers are not used in mixer files.
2. What each of the mixers are for
3. Order and number of outputs - how they work.
4. That detailed definitions are further down.

### CODE REVIEW

File: en/concept/mixing.md [hunk 1]
```diff
@@ -199,35 +202,46 @@ See above for more information on [mixer loading](#loading_mixer).
 
 ### Syntax {#mixer_syntax}
 
-Mixer definitions are text files; lines beginning with a single capital letter followed by a colon are significant.
-All other lines are ignored, meaning that explanatory text can be freely mixed with the definitions.
+Mixer files are text files that define one or more mixer definitions: mappings between one or more inputs and one or more outputs. 
+
+There are four types of mixers definitions: [multirotor mixer](#multirotor_mixer), [helicopter mixer](#helicopter_mixer), [summing mixer](#summing_mixer), and [null mixer](#null_mixer).
+- [Multirotor mixer](#multirotor_mixer) - Defines outputs for 4, 6, or 8 rotor vehicles with + or X geometry.
```
Comment (bkueng) [inline en/concept/mixing.md:208]: There's more flexibility here, e.g. tricopters, or prop-in vs. prop-out. Do we have docs for the mixer geometry files under https://github.com/PX4/Firmware/tree/master/src/lib/mixer/MultirotorMixer/geometries?

**Review by bkueng — APPROVED** [review]

File: en/concept/mixing.md (hunk 1 above)
Comment (hamishwillee) [reply en/concept/mixing.md:208]: @bkueng No docs on geometry files - I'm still hoping @jlecoeur can provide the documentation: https://github.com/PX4/Devguide/issues/349 :-)

I think perhaps given that Julien posted that in 2017 we should perhaps work on that now. Can you help? Specifically, what is geometry file, how do you create it, how does it then relate to the mixer. Let's continue discussion on #349?

Once that is understood perhaps I can add tricopters to this list here.

For now, I plan to merge this.

Comment (bkueng) [reply en/concept/mixing.md:208]: Ah right. Ok if Julien does not reply, ping me again and I can probably fill in the gaps.

Comment (jlecoeur) [reply en/concept/mixing.md:208]: Oops, it is high time to get this done

Comment (hamishwillee) [reply en/concept/mixing.md:208]: @jlecoeur Was that a commitment to getting this done at a particular time? :-) Or do I need to try get Beat to help fix this?

**Pull request merged**